### ISSUE TITLE
Enhance accessibility documentation and measures on Storefront

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2543]
Scope
---
The current documentation of accessibility topics, tests, measures and features on the Storefront is very reduced. The documentation should clearly expose all QA measures and further guidance to enhance and improve accessibility.

Further should every component describe, what kind of accessibility support is provided (e.g. Support for screen reader with "no", "partial", "full").

Notes  
---
- A11y docs on component level: https://www.duetds.com/components/multiselect/#accessibility
- Good example of Wahington Post DS: https://build.washingtonpost.com/resources/accessibility
- Use ux.porsche.com (Frontify CMS) for advanced docu/guidance with images and link from designsystem.porsche.com
- Use designsystem.porsche.com for component specific guidance
- Current component a11y examples should be moved to dedicated a11y tab on component level
- As soon as one component page was refactored it shall be presented and discussed with the team before proceeding with other pages

Acceptance criteria
---
- [x] The Storefront represents a detailed documentation and further guidance of accessibility QA measures, tests and features

Subtasks
---
- [ ] Task
